pkg/migration/migrations/separated_intents.go

```diff
@@ -66,6 +68,7 @@ const migrateLockTableRetries = 3
 type migrateLockTableRequest struct {
 	start, end   roachpb.Key
 	debugRangeID roachpb.RangeID
+	rangeStart   roachpb.RKey // for progress.
```

Comment:
I don't see this set anywhere, also why not reuse `start`, as that's the one we modify when we get a `resumeSpan` too?

Reply:
start here was the actual key we were sending the requests to -- i.e. sometimes the local, sometimes the global. rangeStart here was the key for the range, i.e. what we'd seek to.